PowerEditor/installer/nativeLang/german.xml

```diff
@@ -1109,10 +1109,13 @@ Translation note:
 					<Item id="6506" name="Deaktiviert:"/>
 					<Item id="6507" name="Sprachmenü verschachtelt darstellen"/>
 					<Item id="6508" name="Sprachmenü"/>
-					<Item id="6301" name="Tabulatoren"/>
-					<Item id="6302" name="Ersetzt durch Leerzeichen"/>
-					<Item id="6303" name="Tabulatorbreite:"/>
+					<Item id="6301" name="Einstellung Einrückungen"/>
+					<Item id="6302" name="Leerzeichen"/>
+					<Item id="6303" name="Einrückungsbreite:"/>
+					<Item id="6310" name="Einrückung verwendet:"/>
+					<Item id="6311" name="Tab Zeichen"/>
 					<Item id="6510" name="Standard verwenden"/>
+					<Item id="6512" name="Rückschritttaste entfernt Einrückungen, anstatt einzelne Leerzeichen zu entfernen"/>
```

Comment:
Hi @schnurlos, since "Rückschritttaste" is quite uncommon, albeit absolutely correct, how does the ⌫ ([U+232B](https://en.wikipedia.org/wiki/Miscellaneous_Technical#Block)) character fit the translation? I know, it is not universally accepted, but it might help understanding which key is actually meant.

Something like

> Rückschrittaste ⌫ entfernt Einrückungen statt einzelne Leerzeichen

Reply:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15353 - sorry for the delay.